compiler/rustc_session/src/options.rs

```diff
@@ -2180,6 +2180,8 @@ options! {
         "Use WebAssembly error handling for wasm32-unknown-emscripten"),
     enforce_type_length_limit: bool = (false, parse_bool, [TRACKED],
         "enforce the type length limit when monomorphizing instances in codegen"),
+    experimental_default_bounds: bool = (false, parse_bool, [TRACKED],
```

Comment:
please add a tracking issue for this

Reply:
https://github.com/rust-lang/rust/issues/138781